src/lib/tzcode/zone-abbrevs.txt

```diff
@@ -0,0 +1,344 @@
+A      +01:00          # Alpha Time Zone (Military)
```

Comment:
This is original list used for generation of `timezones.h` by `gen-zone-abbrevs.pl`. It might be reused later for regeneration, but update mode is yet to be created in this (`gen-zone-abbrevs.pl`) script.
https://github.com/tarantool/tarantool/pull/7007/files#diff-30861c316ec4d2c0ae78ff23f574d83f0f4199e4cbfcc371a4fe9c20c67faedcR90

Reply:
Ah, I see now. Thanks!